A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -24,6 +33,10 @@ class CDUInitPage {
                     flightNo = SimVar.GetSimVarValue("ATC FLIGHT NUMBER", "string", "FMC") + "[color]cyan";
                 }
 
+                requestEnable = false;
+                requestButtonLabel = "";
+                requestButton = "";
+
```

Comment:
Because of the above if statement, whenever a pilot tries to turnaround they won't get the option to request a new simbrief route. 

A little hacky method would be it to allow them to enter `SIMBRIEF` into the co-route(like a cheat code), which enables a dummy boolean, if this dummy boolean is true then reenable the requestEnable boolean.

However, to not ruin the immersion, the actual text won't be presented under the COROUTE line. It simply enables a dummy boolean (under the curtains so to speak). 

Once the user has requested the simbrief then the dummy boolean is set to false and thus the text is cleared as per normal with the above if statement in lines 24-25

Reply:
Yes, I agree. I dont know exactly how its done on real life when the flight ends if the initialization is reset. Im looking for pilots feedback but maybe this should be a good workaround

Reply:
The INIT/REVIEW page allows a new flight to be pulled down, overwriting whatever was there before - could you detect an INIT DATA REQ on the INIT/REVIEW page changing dep/dest airports and clear the flight plan in that case? I can't think of a use case for where you'd want to update that page without also updating the flight plan, and it would probably make the behaviour easier to discover...

Reply:
@pareil6 makes sense. That way it resolves two issues at the same time! Great